src/main.rs

```diff
@@ -390,13 +404,16 @@ fn main() {
     info!("Selecting the Chawathe et al. (1996) edit script generator.");
     // Edit script generator configuration object.
 
-    let generator_config: Box<edit_script::EditScriptGenerator<String>> = generator_script_config(args.flag_edit); //  Box::new(edit_script::Chawathe96Config::new());
-                                                                                                                   // This will check the args and compare it with the enum
-                                                                                                                   // There are only 2 enum
-                                                                                                                   // 1) Chawathe96Config
-                                                                                                                   // 2) Chawathe98Config
-                                                                                                                   //
-    fn generator_script_config(input: Option<EditScriptGenerator>) -> Box<edit_script::EditScriptGenerator<String>> {
+    let generator_config: Box<edit_script::EditScriptGenerator<String>> =
+        generator_script_config(args.flag_edit); //  Box::new(edit_script::Chawathe96Config::new());
+    // This will check the args and compare it with the enum
```

Comment:
OK, I think we can kill this comment now, because it just describes what the code does.